public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -1,24 +1,22 @@
 // Libraries
-import cloneDeep from 'lodash/cloneDeep';
-import throttle from 'lodash/throttle';
-import { Subject, Unsubscribable, PartialObserver } from 'rxjs';
-
-// Services & Utils
-import { getDatasourceSrv } from 'app/features/plugins/datasource_srv';
-import kbn from 'app/core/utils/kbn';
-import templateSrv from 'app/features/templating/template_srv';
-import { PanelQueryState } from './PanelQueryState';
-
 // Types
 import {
-  PanelData,
   DataQuery,
-  TimeRange,
-  ScopedVars,
   DataQueryRequest,
   DataSourceApi,
   DataSourceJsonData,
+  PanelData,
+  ScopedVars,
+  TimeRange,
 } from '@grafana/ui';
+import kbn from 'app/core/utils/kbn';
+// Services & Utils
+import { getDatasourceSrv } from 'app/features/plugins/datasource_srv';
+import templateSrv from 'app/features/templating/template_srv';
+import cloneDeep from 'lodash/cloneDeep';
+import throttle from 'lodash/throttle';
+import { PartialObserver, Subject, Unsubscribable } from 'rxjs';
+import { PanelQueryState } from './PanelQueryState';
```

Comment:
can you remove the plugin that scrambles the import order? This make the imports all messy